File: CPP/pallindrome.cpp
#include<iostream>
using namespace std;

int main()
{
   int num=12321, temp,reverse=0,rem;
   temp=num;
   while(temp!=0){
    rem=temp%10;
    reverse=reverse*10 + rem;  
    temp=temp/10; 
   }

   if(num==reverse)
   { 
    cout<<" pallindrome";
   }
   else
   cout<< " not pallindrome";
   return 0;
}

File: Python/py prgs/pallindrome.py
def isPallindrome(num):
    rev=0
    temp=num
    while num>0:
        digit=num%10
        rev=rev*10 + digit
        num=num//10
    return rev==temp

num=int(input('enter number: '))
if isPallindrome(num)==True:
    print(num,' is a pallindrome number')
else:
    print(num,' is not a palindrome number')


